quantum/sequencer/tests/rules.mk

```
@@ -0,0 +1,6 @@
+sequencer_DEFS := -DNO_DEBUG
+
+sequencer_SRC := \
```

And by the following, I meant this:

```suggestion
SEQUENCER_DEFS := -DNO_DEBUG

SEQUENCER_SRC := \
```

---

Ah, that’s because I followed [these Unit Testing instructions](https://docs.qmk.fm/#/unit_testing?id=adding-tests-for-new-or-existing-features). This is how variables are formatted in `quantum/serial_link`, but it indeed seems like an exception when I look at the other `rules.mk` in the codebase. I’ll fix it!

---

I never actually noticed the unit tests used a different case than the rest of the codebase. Good call, disregard my change request.